configuration/locations/locations.csv

```diff
@@ -108,3 +108,24 @@ Sante Mantal | SSPE,b98f6fcd-d3a1-499f-9e26-0424eca5e9ad,,Sante Mantal | SSPE,Sa
 Sante Fanm | SSPE,2cc42f77-6ab9-423b-9f4f-ec61a0fcb55e,,Sante Fanm | SSPE,Saint-Marc (SSPE),,
 Ijans | SSPE,4f88beb9-ad37-44c7-bb89-e7f24002461b,,Ijans | SSPE,Saint-Marc (SSPE),,
 Klinik MNT | SSPE,8d40dd98-7dff-465d-899a-6158b6829d35,,Klinik MNT | SSPE,Saint-Marc (SSPE),,
+COVID-19 | Izolman | La Colline,d02fdde3-2851-4d7e-b34d-fb2dc62aa091,,COVID-19 Isolation at La Colline,La Colline,,
```

Comment:
Basically, I copied all the Thomonde locations and then 1) assigned new uuids to each, and 2) replaced "Thomonde" with "La Colline".  (The main location is named "La Colline", not "Lacolline", we can correct this if this incorrect?)